### ISSUE TITLE
model name not found

### ISSUE THREAD

**Issue by JoOkuma** [kreshuklab/go-nuclear#4]
hey @qin-yu 

I got this error when trying to use the plantseg config at this line 
https://github.com/kreshuklab/go-nuclear/blob/12caacbabca35a3d386fe038afa8fd47cc61f495/plantseg/configs/config_pred_wide_applicability.yaml#L40

```python
RuntimeError: key: model_name has got value: generic_plant_nuclei_3D, but value must be one of ['generic_confocal_3D_unet', 'generic_light_sheet_3D_unet', 'confocal_3D_unet_ovules_ds1x', 'confocal_3D_unet_ovules_ds2x', 'confocal_3D_unet_ovules_ds3x', 'confocal_2D_unet_ovules_ds2x', 'lightsheet_3D_unet_root_ds1x', 'lightsheet_3D_unet_root_ds2x', 'lightsheet_3D_unet_root_ds3x', 'lightsheet_2D_unet_root_ds1x', 'lightsheet_3D_unet_root_nuclei_ds1x', 'lightsheet_2D_unet_root_nuclei_ds1x', 'confocal_2D_unet_sa_meristem_cells', 'confocal_3D_unet_sa_meristem_cells', 'lightsheet_3D_unet_mouse_embryo_cells', 'confocal_3D_unet_mouse_embryo_nuclei', 'PlantSeg_3Dnuc_platinum']
```

I assumed `PlantSeg_3Dnuc_platinum` is the generalist model.

**Comment by qin-yu** [comment]
Sorry @JoOkuma, I somehow completely overlooked all the issues and PRs for this repo. I might have accidentally clicked "unwatch" several months ago. 

You’re absolutely right: `PlantSeg_3Dnuc_platinum` is the model name we ultimately decided on. Initially, I proposed using `generic_plant_nuclei_3D` to align with the naming conventions of PlantSeg models, but we finalized the new name for the manuscript.

Feel free to submit a PR if you'd like! Otherwise, I’ll make the changes in a few days.
